seedfarmer/cli_groups/_bootstrap_group.py

```diff
@@ -54,7 +54,8 @@ def bootstrap() -> None:
 @click.option(
     "--trusted-principal",
     "-t",
-    help="ARN of Principals trusted to assume the Toolchain Role",
+    help="""ARN of Principals trusted to assume the Toolchain Role.
```

Comment:
can you make this more clear about it being valid in single account structure (--as-target)?

Reply:
that doesn't apply to Principals on the Toolchain Role...disregarding this comment